src/main.ts

```diff
@@ -26,7 +26,6 @@ async function run(): Promise<void> {
     const details = await jiraConnector.getTicketDetails(issueKey);
     await githubConnector.updatePrDetails(details);
   } catch (error) {
-    console.log('JIRA key was not found');
```

Comment:
please don't remove this output, it's needed for the case when token is ok, just the jira task not exisitng

Reply:
Oh I see, my mistake